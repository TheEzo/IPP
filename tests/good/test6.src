.IPPcode18
DEFVAR GF@counter
MOVE GF@counter string@
LABEL while
JUMPIFEQ end GF@counter string@aaa
WRITE string@counter\032obsahuje\032
WRITE GF@counter
WRITE string@\010
CONCAT GF@counter GF@counter string@a
JUMP while
LABEL end
STRLEN GF@counter string@ahoj
WRITE GF@counter
defvar GF@cislo
move GF@cislo int@10
add GF@cislo GF@cislo int@1
write GF@cislo
idiv GF@cislo int@10 int@2
write GF@cislo
