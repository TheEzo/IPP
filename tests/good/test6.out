counter obsahuje 



counter obsahuje 
a


counter obsahuje 
aa


4
11
5
